recipes/mkl_random/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.0.1" %}
+{% set buildnumber = 0 %}
+
+
+package:
+    name: mkl_random
+    version: {{ version }}
+
+source:
+    url: https://github.com/IntelPython/mkl_random/archive/v{{version}}.tar.gz
+    md5: f37e16fd040cb28c6cfc726aa68cd8fd
+
+build:
+    number: {{buildnumber}}
+    skip: True  # [win and py27]
```

Comment:
Why is Windows Python 2.7 skipped?

Reply:
The issue here is that Python 2.7 forces specific choice of compiler version, which on Windows does not support c++11 standard. The package uses C++11 feature of standard library, namely sorting with a comparator, see [mkl_distributions.cpp#L623](https://github.com/IntelPython/mkl_random/blob/master/mkl_random/src/mkl_distributions.cpp#L623).

It's not too hard to write a custom fall-back code but it may have inferior performance.

Reply:
No worries. That is not necessary. Was just looking for confirmation that this doesn't work for pre-C++11 compilers.